foundation-extensions/experience-platform-extension/edge-network-api-reference.md

```diff
@@ -279,6 +352,173 @@ public class EdgeEventHandle {
+    * @return instance of current builder
+    * @throws UnsupportedOperationException if this instance was already built
+    */
+  public Builder setXdmSchema(final Map<String, Object> xdm) {...}
+/**
+    * Builds and returns a new instance of {@code ExperienceEvent}.
+    *
+    * @return a new instance of {@code ExperienceEvent} or null if one of the required parameters is missing
+    * @throws UnsupportedOperationException if this instance was already built
+    */
+  public ExperienceEvent build() {...}
+}
+```
+See [Edge Extension Usage](https://github.com/adobe/aepsdk-edge-ios/blob/main/docs/ExtensionUsage.md) for more examples.
+
+{% endtab %}
+
+{% tab title="iOS (AEP 3.x)" %}
+
+### Swift
+
+Experience Event is the event to be sent to Adobe Experience Edge Network.
+The XDM data is required for any Experience Event being sent using the Edge extension.
+
+```swift
+@objc(AEPExperienceEvent)
+public class ExperienceEvent: NSObject {
+
+    /// XDM formatted data, use an `XDMSchema` implementation for a better XDM data injestion and format control
+    @objc public let xdm: [String: Any]?
+
+    /// Optional free-form data associated with this event
+    @objc public let data: [String: Any]?
+
+    /// Optional free-form query data associated with this event
+    @objc public var query: [String: Any]?
+
+    /// Adobe Experience Platform dataset identifier, if not set the default dataset identifier set in the Edge Configuration is used
+    @objc public let datasetIdentifier: String?
+
+    /// Initialize an Experience Event with the provided event data
+    /// - Parameters:
+    ///   - xdm:  XDM formatted data for this event, passed as a raw XDM Schema data dictionary.
+    ///   - data: Any free form data in a [String : Any] dictionary structure.
+    ///   - datasetIdentifier: The Experience Platform dataset identifier where this event should be sent to; if not provided, the default dataset identifier set in the Edge configuration is used
+    @objc public init(xdm: [String: Any], data: [String: Any]? = nil, datasetIdentifier: String? = nil) {
+        self.xdm = xdm
+        self.data = data
+        self.datasetIdentifier = datasetIdentifier
+    }
+
+    /// Initialize an Experience Event with the provided event data
+    /// - Parameters:
+    ///   - xdm: XDM formatted event data passed as an XDMSchema
+    ///   - data: Any free form data in a [String : Any] dictionary structure.
+    public init(xdm: XDMSchema, data: [String: Any]? = nil) {
+        if let jsonXdm = xdm.toJSONData() {
+            self.xdm = try? JSONSerialization.jsonObject(with: jsonXdm, options: []) as? [String: Any]
+        } else {
+            self.xdm = nil
+        }
+        self.data = data
+        self.datasetIdentifier = xdm.datasetIdentifier
+    }
+}
+```
+See [Edge Extension Usage](https://github.com/adobe/aepsdk-edge-ios/blob/main/docs/ExtensionUsage.md) for more examples.
```

Comment:
I think we can remove this link from each section/tab and reference it only once after all tabs were closed.